src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -67,6 +67,16 @@
  * <pre>
  * &lt;module name=&quot;SingleSpaceSeparator&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo()   { //violation, 3 whitespaces
+ *   return 1+2  ; //violation, 2 whitespaces
```

Comment:
`1+2` please change to one digit.
Also, I think it is more sensible to make violation between return and value, not value and semicolon